config/clusters/leap/prod.values.yaml

```diff
@@ -6,6 +6,7 @@ basehub:
     singleuser:
       extraEnv:
         SCRATCH_BUCKET: gcs://leap-scratch/$(JUPYTERHUB_USER)
+        PANGEO_SCRATCH: gcs://leap-scratch/$(JUPYTERHUB_USER)
```

Comment:
```suggestion
        # Pangeo defines this variable in their image, so we also use it here+staging to avoid confusion.
        PANGEO_SCRATCH: gcs://leap-scratch/$(JUPYTERHUB_USER)
```